src/views/ActivityDaily.vue

```diff
@@ -385,13 +404,26 @@ export default {
 
     queryWindows: async function() {
       var periods = [this.dateStart + "/" + this.dateEnd];
-      var q = query.windowQuery(this.windowBucketId, this.afkBucketId, this.top_apps_count, this.top_windowtitles_count, this.filterAFK);
+      let classes = [
+        ["Media -> Games", "RimWorld"],
+        ["Media -> Video", "YouTube"],
+        ["Media -> Social Media", "reddit|Facebook|Twitter"],
+        ["Remote", "Shadow"],
+        ["Work", "[Aa]lacritty|Google Docs"],
+        ["Work -> Programming", "[Pp]ython|GitHub"],
+        ["Work -> Programming -> ActivityWatch", "aw-|[Aa]ctivity[Ww]atch"],
+        ["Comms -> IM", "Messenger"],
+        ["Comms -> Email", "Gmail"],
```

Comment:
Don't use "->" and split the string, use arrays instead and pretty-print later with -> if that's what you want to do.